Source/Models/BrickDimension.swift

```diff
@@ -27,6 +27,7 @@ extension UIView {
 public indirect enum BrickDimension {
     case Ratio(ratio: CGFloat)
     case Fixed(size: CGFloat)
+    case Remainder
```

Comment:
Would this be able to take the place of a `Ratio(ratio: 1.0)` dimension? For example, if you called this `Fill` instead of `Remainder`, it would cover cases of a 100% ratio brick AND a "remainder" brick. Just a thought.